Extension/tslint.json

```diff
@@ -1,31 +1,122 @@
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
         "curly": true,
+        "encoding": true,
+        "eofline": true,
         "ext-variable-name": [
             true,
-            ["class", "pascal"],
-            ["function", "camel"]
+            [
+                "class",
+                "pascal"
+            ],
+            [
+                "function",
+                "camel"
+            ]
         ],
-        "file-header": [true, ".*"],
-        "indent": [true, "spaces", 4],
+        "file-header": [
+            true,
+            ".*"
+        ],
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "label-position": true,
+        "match-default-export-name": true,
+        "member-ordering": true,
         "new-parens": true,
+        "no-arg": true,
+        "no-bitwise": true,
+        "no-boolean-literal-compare": true,
+        "no-conditional-assignment": true,
         "no-consecutive-blank-lines": true,
         "no-construct": true,
+        "no-debugger": true,
+        "no-default-export": true,
+        "no-duplicate-imports": true,
+        "no-duplicate-super": true,
+        "no-duplicate-switch-case": true,
         "no-duplicate-variable": true,
         "no-unused-expression-chai": true,
+        "no-eval": true,
+        "no-import-side-effect": true,
+        "no-internal-module": true,
+        "no-invalid-this": true,
+        "no-irregular-whitespace": true,
+        "no-mergeable-namespace": true,
+        "no-misused-new": true,
+        "no-namespace": true,
+        "no-non-null-assertion": true,
+        "no-redundant-jsdoc": true,
+        "no-reference": true,
+        "no-reference-import": true,
+        "no-return-await": true,
+        "no-sparse-arrays": true,
+        "no-switch-case-fall-through": true,
+        "no-this-assignment": true,
+        "no-trailing-whitespace": true,
+        "no-unnecessary-callback-wrapper": true,
+        "no-unnecessary-initializer": true,
+        "no-unnecessary-qualifier": true,
+        "no-unsafe-finally": true,
+        "no-unused-expression": true,
         "no-unused-variable": true,
         "no-var-keyword": true,
-        "one-line": [true, "check-catch", "check-finally", "check-else", "check-open-brace", "check-whitespace"],
+        "no-var-requires": true,
+        "number-literal-format": true,
+        "one-line": [
+            true,
+            "check-catch",
+            "check-finally",
+            "check-else",
+            "check-open-brace",
+            "check-whitespace"
+        ],
+        "one-variable-per-declaration": true,
+        "prefer-method-signature": true,
+        "prefer-object-spread": true,
+        "prefer-while": true,
         "promise-must-complete": true,
         "semicolon": true,
+        "space-within-parens": true,
```

Comment:
Not sure if this is doing anything:
https://palantir.github.io/tslint/rules/space-within-parens/

Reply:
No, this is correct. The docs are wrong. Or...it's correct with my version of TS Lint, which enforces the 0 space rule when "true" is used and doesn't accept a number like the docs say.